lidar_topic: "/rslidar_points"
imu_topic: "/imu/data"


lidar_type: "velodyne"

extrinsic:
  rotation: [0.0, 1.0, 0.0, 0.0] # Quaternion [w,x,y,z]
  translation: [ 0.065, 0.0, -0.05 ] # Translation vector


publish_voxmap: true





